lib/internal/test_runner/reporter/lcov.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const Transform = require('internal/streams/transform');
+
+// This reporter is based on the LCOV format, as described here:
+// https://ltp.sourceforge.net/coverage/lcov/geninfo.1.php. Excerpts from this
```

Comment:
This dot is infered as part of the URL in the IDE (and browser) so I think best to remove it?
```suggestion
// https://ltp.sourceforge.net/coverage/lcov/geninfo.1.php Excerpts from this
```